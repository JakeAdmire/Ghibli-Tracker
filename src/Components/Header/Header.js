import React from 'react';

export function Header() {

  return (
    <div className="Header">
      <h1>GhibliTracker</h1>
    </div>
  )

}