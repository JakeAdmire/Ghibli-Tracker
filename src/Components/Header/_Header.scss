.Header {
  background: grey;
  width: 100%; height: 80px;
  position: fixed;
  top: 0;
  left: 0;
  right: 0;

  @include flexBox(row, center, center);

  h1 {
    margin: 0;
    
    @include customFont(50px, initial, initial, #fff);
  }
}