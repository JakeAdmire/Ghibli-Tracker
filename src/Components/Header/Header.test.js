import React from 'react';
import { Header } from './Header';
import { shallow } from 'enzyme';

describe('Header', () => {

  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Header/>)
  })

  it('should match the snapshot', () => {
    expect(wrapper.debug()).toMatchSnapshot();
  })

})