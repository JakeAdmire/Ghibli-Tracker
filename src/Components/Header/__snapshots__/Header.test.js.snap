// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match the snapshot 1`] = `
"<div className=\\"Header\\">
  <h1>
    Howl&#39;s Movie Castle
  </h1>
</div>"
`;
